clients/index.js

```diff
@@ -479,3 +480,16 @@ ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches()
         }
     }
 };
+
+ApplicationClients.prototype.updatePeerHeapEnabled = function updatePeerHeapEnabled() {
+    var self = this;
+    var peerHeapConfig = self.remoteConfig.get('peer-heap.enabled.services', {});
+    var keys = Object.keys(peerHeapConfig);
+    for (var i = 0; i < keys.length; i++) {
+        var serviceName = keys[i];
+        var enabled = peerHeapConfig[serviceName];
+        if (self.channel.subChannels[serviceName]) {
```

Comment:
This is not correct. ServiceProxy will allocate sub channels dynamically.

Please call a method on ServiceProxy and do it properly there.